scripts/game_controller.js

```diff
@@ -0,0 +1,41 @@
+// game_controller.js
+
+const { getPlayerGame, isPositionFree, getGame, playerDidPlay } = require('./game.js');
+const { PLAYER_1, PLAYER_2 } = require('./players.js');
+
+function play(player, position) {
+    playerDidPlay(player, position);
+    return {'GAME': getGame()};
+} 
+
+// Exports
+
+const nextPlayer = () => {
+    const firstPlayerGame = getPlayerGame(PLAYER_1);
+    const secondPlayerGame = getPlayerGame(PLAYER_2);
+
```

Comment:
I think the easiest way to do this check is just to see if the total number of plays is odd or even:

`((firstPlayerGame.length + secondPlayerGame.length ) % 2) === 0 ? PLAYER_1 : PLAYER_2 `

Reply:
Oh nice! Thanks Sam, will update it 😄